README.md

```diff
@@ -19,21 +16,36 @@ In this workshop we'll be building on what we learnt in the [pg walkthrough](htt
 - Create a superuser with a password by typing `CREATE USER [the new username] WITH SUPERUSER PASSWORD '[the password of the database]';` (the password needs to be in quotes, otherwise you get an error).
 - Change ownership of the database to the new user by typing `ALTER DATABASE [name of the database] OWNER TO [the new username];`
 - Add a `config.env` file and add the database's url in this format:
-`DB_URL = postgres://[username]:[password]@localhost:5432/[database]`.
+`DB_URL = postgres://[username]:[password]@localhost:5432/[database]`. The database name needs to be in lower case.
 
 ### Task 1
 
-In `src/handler.js` you'll find a function that deals with calls to the `/users` endpoint. The data is currently coming from the `static.js` file. You can change the data in `static.js` and check the results.
+In `src/router.js` you'll find a function that deals with calls to the `/users` endpoint. The data is currently coming from the `handlers/static.js` file. You can change the data in `handlers/static.js` and check the results.
 
-Your job is to refactor the handler so that the response data comes from the users table in your database instead of from `static.js`.
+In the terminal, connect to your database using `psql postgres://[username]:[password]@localhost:5432/[database]`.
 
-### Task 2
+Create a table called 'users' with three columns: 'id', 'name' and 'location' and add a couple of rows of dummy data.
 
-So far, we've only been dealing with `GET`ting data from a server. But what if we want to send some new data to our database instead?
+**Hint: don't hard code the ids**
+
+Your job is to add to `database/db_connection.js` and `handlers/getData.js` and refactor `router.js` so that the response data comes from the users table in your database instead of from `handlers/static.js`.
+
+Once you've done this, move onto task 2.
+
+### Task 2
+** CSS Challenge! **
 
 - Open up `index.html` and uncomment the form
 - Check that you can see an empty input box when you refresh your browser
-- Create a new handler function for the endpoint `create-user`. What kind of **method** do you think you'll need? How will you access the data from your form?
+- Try and get the design looking as close as possible to the below before moving on:
```

Comment:
The old Task 2 had students un-commenting-out the form, because this was _directly_ related to the `create-user` handler they were creating.

Having the CSS task in the middle of your database tasks feels pretty random. The tasks don't flow together at all.